src/main/resources/application-context.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+	   					   http://www.springframework.org/schema/beans/spring-beans.xsd">
+
+    <!--bean id="callbackFilter" class="org.pac4j.j2e.filter.CallbackFilter">
+        <property name="clientsFactory" value="org.pac4j.demo.j2e.config.DemoClientsFactory" />
+        <property name="defaultUrl" value="/" />
+    </bean-->
+
+    <bean id="customAuthorizer" class="org.pac4j.demo.j2e.authorizer.CustomAuthorizer" />
```

Comment:
Using dependency injection to create the custom authorizer and inject it in a `RequiresAuthenticationFilter`.